docs/layout.pug

```diff
@@ -143,6 +143,8 @@ html(lang='en')
                 a.pure-menu-link(href="/docs/6.x/docs/migrating_to_6.html", class=outputUrl === '/docs/6.x/docs/migrating_to_6.html' ? 'selected' : '') Migration Guide
               li.pure-menu-item
                 a.pure-menu-link(href="/docs/6.x/docs/compatibility.html", class=outputUrl === '/docs/6.x/docs/compatibility.html' ? 'selected' : '') Version Compatibility
+              li.pure-menu-item
+                a.pure-menu-link(href="/docs/6.x/docs/version-support.html", class=outputUrl === '/docs/6.x/docs/version-support.html' ? 'selected' : '') Version Support
```

Comment:
no entry for `check-version.md`?

Reply:
No, we'll just put that one in the guides page. I'd like to rethink our navbar, because right now there isn't really a good place to put check-version.md